In JavaScript you can pass an error object to `captureException()` to get it captured
as event. It's possible to throw strings as errors in which case no traceback
can be recorded.

```javascript
try {
  aFunctionThatMightFail();
} catch (err) {
  Sentry.captureException(err);
}
```

Learn more about how to manually capture errors or enable message capture with Sentry's SDK in <PlatformLink to="/usage/">our documentation on Usage</PlatformLink>.
